smap/sources/aqiDriver.py

```diff
@@ -0,0 +1,34 @@
+from smap.driver import SmapDriver
+from smap.util import periodicSequentialCall
+from smap.contrib import dtutil
+from aqiCalculator import init_aqi_drivers, get_aqi_data
+
+# logging
+import sys, traceback
+import logging
+logging.basicConfig(level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(message)s')
+
+class AQIDriver(SmapDriver):
+	"""
+	This Driver computes hourly average of data from Dylos Sensors and converts it to AQI and posts it to server
+	"""
+	def setup(self, opts):
+		self.rate = int(opts.get('Rate', 3600))
+		self.dylosSensors = init_aqi_drivers()
+		for key, value in self.dylosSensors.iteritems():
+			dylosAQIDriver = self.add_timeseries("/"+key,'AQI',description='Air Quality Index - India')
+			dylosAQIDriver['Properties']['Timezone'] = 'Asia/Kolkata'
+
+	def start(self):
+		self.process = periodicSequentialCall(self.read)
+		self.process.start(self.rate)
+
+	def read(self):
```

Comment:
It is because in driver conf file, we have defined the report0 location to which the driver sends the data to be added.